velox/exec/fuzzer/MemoryArbitrationFuzzer.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cstddef>
+
+namespace facebook::velox::exec::test {
+void memoryArbitrationFuzzer(size_t seed);
```

Comment:
not sure how other fuzzers do. Shall we expose .go() method to the caller? This method name indicates it is going to return a fuzzer object, but instead it calls go() method to start the fuzzer already.